nservicebus/sqlserver/multiple-databases.md

```diff
@@ -14,24 +14,48 @@ The SQL Server transport allows you to select, on per-endpoint basis, where the
+The default schema used by SQL Server transport is `dbo`. To specify a different schema use the following API:
 
+snippet:sqlserver-singledb-multischema
 
-## Database-per-endpoint with store-and-forward
+or using a configuration file:
+
+snippet:sqlserver-singledb-multischema-config
+
+If two endpoints use different schemas then additional configuration is required. The sender needs to know the schema of the receiver, and subscriber needs the schema of the publisher. 
+
+The schema for another endpoint can be specified in the following ways:
+
+snippet:sqlserver-singledb-multidb-push
+
+or:
+
+snippet:sqlserver-singledb-multidb-pull
+
+NOTE: Even if two endpoints use different schemas the SQL Server transport will assume they use the same database. The different connection string must be passed explicitly, as described in the following section.
+
+## Multiple databases
+
+Endpoints can also use separate databases. That scenario requires DTC. 
+
+NOTE: Due to the lack of store-and-forward mechanism, if a remote endpoint's database or DTC infrastructure is down, the endpoint cannot send messages to it. This potentially renders the endpoint unavailable (and also all other endpoints depending on it directly or indirectly).
+
+## Multiple databases with store-and-forward
 
 In order to overcome this limitation a higher level store-and-forward mechanism needs to be used. The Outbox feature can be used to effectively implement a distributed decoupled architecture where:
  * Each endpoint has its own database where it stores both the queues and the user data
- * Messages are not sent immediately when calling `Bus.Send()` but are added to the *outbox* that is stored in the endpoint's own database. After completing the handling logic the messages in the *outbox* are forwarded to their destination databases
- * Should one of the forward operations fail, it will be retried by means of [standard NServiceBus retry mechanism](/nservicebus/errors/automatic-retries.md). This might result in some messages being sent more than once but it is not a problem because the outbox automatically handles the deduplication of incoming messages based on their ID.
+ * Messages are not sent immediately when calling `Bus.Send()` but are added to the Outbox table that is stored in the endpoint's own database. After completing the handling logic the messages stored in the Outbox table are forwarded to their destination databases
+ * Should one of the forward operations fail, it will be retried by means of the [standard NServiceBus retry mechanism](/nservicebus/errors/automatic-retries.md). This might result in some messages being sent more than once but Outbox automatically handles the deduplication of incoming messages based on their ID, providing `exactly-once` message delivery guarantee.
```

Comment:
I think rather than one of the forward operations, it should be any of the forward operations.

Here's a suggestion:

Should any of the forward operations were to fail, it will be retried using the standard [retry mechanism](/nservicebus/errors/automatic-retries.md). However, this may result in some messages to be sent multiple times. To mitigate this and to provide `exactly-once` message delivery guarantee, the Outbox feature automatically handles the de-duplication of incoming messages based on their ID.